test/factory.js

```diff
@@ -227,4 +229,28 @@ factory.define('sourceRelatedToSeveralRecords', Source, {
   },
 });
 
+factory.define('fda_application', FdaApplication, {
+  id: () => uuid.v1(),
+  drug_name: 'Healer',
+  active_ingredients: 'healing',
+  organisation_id: factory.assoc('organisation', 'id'),
+}, {
+  afterCreate: (fdaApplication, attrs, callback) => {
+    factory.create('fda_approval', { fda_application_id: fdaApplication.id })
+    .then(() => new FdaApplication({ id: fdaApplication.id })
+      .fetch({ withRelated: ['fda_approvals', 'organisation'] }))
+    .then((instance) => callback(null, instance))
+    .catch((err) => callback(err));
```

Comment:
Fix indentation and use the model's `FdaApplication.relatedModels`, so we avoid repetitions